<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>pip on 鬼画桃符</title><link>https://tab.deoops.com/tags/pip/</link><description>Recent content in pip on 鬼画桃符</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Fri, 30 Mar 2018 10:21:25 +0800</lastBuildDate><atom:link href="https://tab.deoops.com/tags/pip/index.xml" rel="self" type="application/rss+xml"/><item><title>pip包冲突</title><link>https://tab.deoops.com/posts/python-import/</link><pubDate>Fri, 30 Mar 2018 10:21:25 +0800</pubDate><guid>https://tab.deoops.com/posts/python-import/</guid><description>遇到一个奇怪的问题执行certbot会报错，moudle conflict和 No module，但是yum install certbot的时候没有报错。 certbot Traceback (most recent</description></item></channel></rss>